<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_addCourse</class>
 <widget class="QDialog" name="dialog_addCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加课程</string>
  </property>
  <property name="styleSheet">
   <string>
     QDialog {
        background-color: #e6f3ff;
        font-family: &quot;Segoe UI&quot;, sans-serif;
     }
   </string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>381</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="styleSheet">
       <string>
         QPushButton {
            background-color: #4CAF50;
            color: white;
            font-size: 16px;
            font-weight: bold;
            border-radius: 5px;
            padding: 10px 20px;
            border: 1px solid #4CAF50;
            min-width: 100px;
         }
         QPushButton:hover {
            background-color: #45a049;
         }
         QPushButton:pressed {
            background-color: #388E3C;
         }
       </string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="styleSheet">
       <string>
         QPushButton {
            background-color: #f44336;
            color: white;
            font-size: 16px;
            font-weight: bold;
            border-radius: 5px;
            padding: 10px 20px;
            border: 1px solid #f44336;
            min-width: 100px;
         }
         QPushButton:hover {
            background-color: #e53935;
         }
         QPushButton:pressed {
            background-color: #d32f2f;
         }
       </string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lable_CNo">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>课程号（必填）</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_CNo">
      <property name="styleSheet">
       <string>
         QLineEdit {
            font-size: 14px;
            padding: 5px;
            border: 1px solid #ccc;
            border-radius: 3px;
         }
       </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_CName">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>课程名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_CName">
      <property name="styleSheet">
       <string>
         QLineEdit {
            font-size: 14px;
            padding: 5px;
            border: 1px solid #ccc;
            border-radius: 3px;
         }
       </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_CTeacher">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>授课教师</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_CTeacher">
      <property name="styleSheet">
       <string>
         QLineEdit {
            font-size: 14px;
            padding: 5px;
            border: 1px solid #ccc;
            border-radius: 3px;
         }
       </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_CClass">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>授课地点</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_CAdder">
      <property name="styleSheet">
       <string>
         QLineEdit {
            font-size: 14px;
            padding: 5px;
            border: 1px solid #ccc;
            border-radius: 3px;
         }
       </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_CTime">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>授课时间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox_CTime">
        <property name="styleSheet">
         <string>
           QComboBox {
              font-size: 14px;
              padding: 5px;
              border: 1px solid #ccc;
              border-radius: 3px;
           }
         </string>
        </property>
        <item>
         <property name="text">
          <string>星期一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期三</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期四</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期五</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期六</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期天</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_CTime">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">           QTimeEdit {
                font-size: 14px;  &lt;!-- 保持与其他文本框相同的字体大小 --&gt;
                padding: 5px;  &lt;!-- 保持一致的内边距 --&gt;
                border: 1px solid #ccc;  &lt;!-- 保持一致的边框颜色 --&gt;
                border-radius: 3px;  &lt;!-- 圆角设置为与其他文本框一致 --&gt;
            }</string>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Credit">
      <property name="styleSheet">
       <string>
         QLabel {
            font-size: 14px;
            font-weight: bold;
            color: #333;
         }
       </string>
      </property>
      <property name="text">
       <string>学分</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_Credit">
      <property name="styleSheet">
       <string>
         QLineEdit {
            font-size: 14px;
            padding: 5px;
            border: 1px solid #ccc;
            border-radius: 3px;
         }
       </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_CNo</tabstop>
  <tabstop>lineEdit_CName</tabstop>
  <tabstop>lineEdit_CTeacher</tabstop>
  <tabstop>lineEdit_CAdder</tabstop>
  <tabstop>lineEdit_Credit</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
